### PR TITLE
Feat/bookings

### PR DESCRIPTION
**Pull request by daveGrayMac**
Please review papi, not pretty but functional. Sends email with all the necessary information. Have added ReCaptcha but not rendering on local host will have to update when live I believe.

### CODE REVIEW

File: components/BookingForm.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+            className="form-control email-inputs"
+            name="user_location"
+            type="text"
+            id="location"
+            onChange={handleFormChange('location')}
+            required
+          />
+        </div>
+        <div>
+          <TextField
```
Comment (djbotha) [inline components/BookingForm.js:144]: Have a look at the [material-ui datepickers](https://material-ui.com/components/pickers/#datepickers), they do all of this for you (and work a looot better than trying to do it yourself).

File: components/ReCaptchaComp.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  render() {
+    return (
+      <div>
+        <ReCaptcha
+            ref={(el) => {this.captcha = el;}}
+            size="normal"
+            data-theme="dark"            
+            data-badge="inline"
+            render="explicit"
+            //Add the site key here
```
Comment (djbotha) [inline components/ReCaptchaComp.js:35]: FYI it is able to run recaptcha on localhost. Just need to configure a few things on google's side.

File: components/BookingForm.js [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+  return (
+    //Form layout that requires a Name, Email, and message
+    <form className={classes.root} onSubmit={handleSubmit}>
+      <br />
+      <div style={{ fontSize: '1.2rem' }}>
+        <h6>You can also send me an email directly from here</h6>
+        <div>
+          <label htmlFor="name">Name</label>
+          <input
```
Comment (andreimid) [inline components/BookingForm.js:107]: @djbotha please confirm, but I think the `value={name}` etc also need to be added to these `<input>` fields

**Review by djbotha — CHANGES_REQUESTED** [review]
That date capture is going to break really quickly (like specifying `December` as the month, the month will be formatted as `012` (or `011`, depends how JS' month counter works).

File: components/BookingForm.js (hunk 3 above)
Comment (djbotha) [reply components/BookingForm.js:107]: Ah yeah, nice catch

File: pages/bookings/gcal.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import request from 'superagent'
+
+const CALENDAR_ID = 'gb3c9vmjaaptk1f5a8okugd2s8@group.calendar.google.com'
```
Comment (andreimid) [inline pages/bookings/gcal.js]: Put these keys in `.env.local` and `.next.config.js`

**Review by andreimid — CHANGES_REQUESTED** [review]
Just one last change to make then I'm happy

**Review by andreimid — APPROVED** [review]

**Review by djbotha — APPROVED** [review]
Muuuch better! Looking good.

**Pull request merged**